packages/adminrouter/extra/src/test-harness/tests/test_agent.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+import grp
+
+import psutil
+import pytest
```

Comment:
@mellenburg What do you think?

The things I care for:
a) we execute `make flake8` also during AR Jenkins job (so we will have some duplication/both tests will go red if something goes wrong)
b) AR rules are bit stricter than general DC/OS rules (line lenght to be precise) - I would like to keep it that way.

I am willing to be convinced when it comes to a), I would very much like to keep b).

Thoughts?